cvat/apps/engine/schema.py

```diff
@@ -228,4 +229,27 @@ class CloudStorageReadSerializerExtension(_CloudStorageSerializerExtension):
 class CloudStorageWriteSerializerExtension(_CloudStorageSerializerExtension):
     target_class = 'cvat.apps.engine.serializers.CloudStorageWriteSerializer'
 
+ORGANIZATION_OPEN_API_PARAMETERS = [
```

Comment:
How and when this should be used? Is it for backward compatibility only?

Reply:
We use this in "schema" decorators for views like [here](https://github.com/opencv/cvat/blob/ed3dbe8f9d711a9894b51977b3ad11f4ec89f374/cvat/apps/engine/views.py#L1724-L1730). But yes I agree probably it would be more logical to move it in iam